cmd/nerdctl/images.go

```diff
@@ -103,13 +104,43 @@ func imagesAction(cmd *cobra.Command, args []string) error {
 	var (
 		imageStore = client.ImageService()
 	)
+	var imageList []images.Image
 
-	// To-do: Add support for --filter.
-	imageList, err := imageStore.List(ctx, filters...)
-	if err != nil {
-		return err
-	}
```

Comment:
this https://github.com/containerd/nerdctl/pull/1307/files#diff-9e27a209503efca44a132d2f31a04fef12646abd8ec11bd3deb9848d4daf745bR123-R126 :  should be done here before entering in the if clause :